techniques/aria/ARIA24.html

```diff
@@ -2,64 +2,55 @@
       <section id="important-information">
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques.html">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>Content using <a href="https://www.w3.org/TR/wai-aria/">Accessible Rich Internet Applications (WAI-ARIA)</a>.
+            <h2>適用 (対象)</h2>
+            <p><a href="https://www.w3.org/TR/wai-aria/">Accessible Rich Internet Applications (WAI-ARIA)</a>を用いたコンテンツ。
             </p>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/info-and-relationships">Success Criterion 1.3.1: Info and Relationships</a> (Sufficient)</span>.
+            <p>これは<span><a href="https://waic.jp/docs/WCAG21/Understanding/info-and-relationships">達成基準  1.3.1: 情報及び関係性を理解する</a></span>  (十分な達成方法) に関する達成方法である。 
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to show how to semantically identify an element
-               that uses a font file for icons. When a user  overrides font-family these icons typically
-               disappear unless there is a means to identify them. The point is to provide a technique
-               to differentiate icon fonts from general font (text) usage.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、アイコンにフォントファイルが使用されている要素を、意味的に識別する方法を示すことである。ユーザーがフォントファミリーを上書きすると、識別する手段がない限りそのアイコンは消える。つまり、アイコンフォントを一般的なフォント（テキスト）使用と区別する手段を提供する。
             </p>
-            <p>Some low vision users need user stylesheets, scripts, or extensions to override the
-               fonts on a page to perceive text content. However, they need to be able to perceive
-               icon fonts that are meaningful, such as a star signifying a favorite, or an email
-               icon in a link.
+            <p>一部の弱視の利用者は、テキストコンテンツを認識するために利用者用のスタイルシート、スクリプト、又は拡張機能を使いページ上のフォントを上書きする。しかし、お気に入りを示す星やリンク内のメールアイコンなど、意味のあるアイコンフォントは認識できる必要がある。
```

Comment:
```suggestion
            <p>一部のロービジョンの利用者は、テキストコンテンツを認識するためにユーザスタイルシート、スクリプト、又は拡張機能を使いページ上のフォントを上書きする必要がある。しかし、お気に入りを示す星やリンク内のメールアイコンなど、意味のあるアイコンフォントは認識できる必要がある。
```

複合語利用者としなくてだいたい大丈夫です